packages/manager/apps/pci-public-ip/src/pages/additional-ips/terminate/TerminateAdditionalIPPage.tsx

```diff
@@ -21,25 +20,33 @@ export default function TerminateAdditionalIPPage() {
     projectId,
     onSuccess: () => {
       addSuccess(
-        t(
-          'pci_additional_ips_floating_ips_floating_ip_terminate_success_info',
-          {
-            ip: failoverIP.ip,
-          },
-        ),
+        <Translation ns="common">
```

Comment:
The Translation component hasn't common a default ns value ?

Reply:
I have updated :)